charts/jaeger-v2/Chart.lock

```diff
@@ -0,0 +1,15 @@
+dependencies:
+- name: cassandra
+  repository: https://charts.helm.sh/incubator
+  version: 0.15.3
+- name: elasticsearch
+  repository: https://charts.bitnami.com/bitnami
+  version: 20.0.4
+- name: kafka
+  repository: https://charts.bitnami.com/bitnami
+  version: 26.6.2
```

Comment:
Does bitnami have different versions for different charts, or is it released as a single version?

Reply:
```
$ helm search repo bitnami/kafka
NAME            CHART VERSION   APP VERSION     DESCRIPTION                                       
bitnami/kafka   30.1.5          3.8.0           Apache Kafka is a distributed streaming platfor...
```
```
$ helm search repo bitnami/elasticsearch
NAME                    CHART VERSION   APP VERSION     DESCRIPTION                                       
bitnami/elasticsearch   21.3.20         8.15.2          Elasticsearch is a distributed search and analy...
```
Kafka Helm Chart
Chart version: 30.1.5
App version: 3.8.0

Elasticsearch Helm Chart
Chart version: 20.0.4
App version: 8.5.0

In this example, you can see that the Kafka chart has its own version (30.1.5), and the Elasticsearch chart has a completely different version (20.0.4). Both have different chart versions because the Helm charts for these applications are developed and released independently.